src/desktop/space/element.rs

```diff
@@ -10,6 +10,33 @@ use std::{
 };
 use wayland_server::protocol::wl_surface::WlSurface;
 
+/// Indicates default values for some zindexs inside smithay
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[repr(u8)]
+pub enum RenderZindex {
+    /// WlrLayer::Background default zindex
+    Background = 10,
+    /// WlrLayer::Bottom default zindex
+    Bottom = 20,
+    /// Default zindex for Windows
+    Shell = 30,
+    /// Default zindex for Windows PopUps
+    PopUpsShell = 40,
+    /// WlrLayer::Top default zindex
+    Top = 50,
+    /// Default Layer for RenderElements
+    Overlay = 80,
+    /// Default Layer for Overlay PopUp
+    PopUpsOverlay = 100,
```

Comment:
Also please stylize popups as "Pop**u**p", not "Pop**U**p, to make it consistent to the Pop**u**pManager and other structs already in smithay.